README.md

```diff
@@ -2,12 +2,38 @@
 
 [![License](https://img.shields.io/badge/License-Apache_2.0-brightgreen.svg)](https://github.com/uclasystem/VQPy/blob/main/LICENSE)
 
-VQPy is an object-oriented language for modern video analytics. With VQPy, users can express their video queries in a composable and reusable manner with Python. 
+VQPy is an object-oriented language for modern video analytics. With VQPy, users can express their video queries in a composable and reusable manner with Python.
 
 VQPy is still under active development. VQPy compiler, which generates a query plan with optimized performance for users' video analytics applications, is working in progress. With VQPy compiler, users can simply focus on the declaration of video queries for their own applications, and multiple optimizations defined in the compiler will be transparently applied to the user’s video analytics pipeline.
 
 ## Getting Started
 
+### Setup
+
+Install PyTorch (>=1.7 as required by YOLOX).
+
```

Comment:
#### Install VQPy dependencies

VQPy use yolox as one of the built in model. 

To install yolox, first we need to install pytorch